app/services/apiEndpoints.js

```diff
@@ -1,2 +1,6 @@
-export const GET_USERS = '/api/users';
-export const GET_USER = '/api/users';
+export const BASE_URL = 'http://localhost:3000/api';
```

Comment:
Thanks! Actually I can leave a TODO here to use a config value for the `BASE_URL` hehe.